README.md

```diff
@@ -88,6 +88,48 @@ postgresql:
 See the configuration section for more options if needed
 Please refer to the Configuration section below on how to enable this.
 
+## Smoke and functional tests
+
+From version 2.14.0 of this chart you can configure your functional and smoke tests to run just after deployment or at scheduled times 
+as cron jobs.
+
+```yaml
+java:
+  testsConfig:
+    keyVaults:
+      "cmc":
+        excludeEnvironmentSuffix: false
+        secretRef: "kvcreds"
+        secrets:
+          smoke-test-citizen-username: SMOKE_TEST_CITIZEN_USER
+          smoke-test-user-password: SMOKE_TEST_CITIZEN_PASS
+    environment:
+      TEST_URL: http://plum-recipe-backend-java
+
+  smoketests:
+    image: hmctspublic.azurecr.io/spring-boot/template-test
+    enabled: true
+    environment:
+      TEST_URL: http://plum-recipe-backend-java-overridden
+
+  functionaltests:
+    image: hmctspublic.azurecr.io/spring-boot/template-test
+    enabled: true
+
+  smoketestscron:
```

Comment:
cron is a separate word, shouldn't we camel or kebab case?
```suggestion
  smoketestsCron:
```

```suggestion
  smoketests-cron:
```

Reply:
Camel causes trouble with dns naming convetions, so it's trouble. Kebab case would be possible, but I like it better in one word to be honest